src/MicroKernel.php

```diff
@@ -28,8 +28,24 @@ public function __construct()
 
     private function initRoutes(): void
     {
-        $this->routes->add('route_name', new Route('/{argument}', ['_controller' => 'App\Controller\MyController']));
-        // Add your Routes here. documentation here https://symfony.com/doc/4.2/components/routing.html
+        //$this->routes->add('route_name', new Route('/', ['_controller' => 'App\Controller\MyController']));
+
+        // Category
+        $this->routes->add('category_find_all', new Route('/categories', ['_controller' => 'App\Controller\Category\FindAllController']));
```

Comment:
trick for the future:
import the class and use it like that instead
```suggestion
        $this->routes->add('category_find_all', new Route('/categories', ['_controller' => FindAllController::class]));
```